test/functional/core/fileio_spec.lua

```diff
@@ -2,6 +2,7 @@ local helpers = require('test.functional.helpers')(after_each)
 
 local assert_log = helpers.assert_log
 local assert_nolog = helpers.assert_nolog
+local write_file = helpers.write_file
```

Comment:
This variable is defined again below, so this line should be removed.

Reply:
thanks